odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -577,6 +584,29 @@ def test_27_company_dependent(self):
         self.assertEqual(attribute_record.bar, 'DEFDEF')
         self.assertFalse(self.env.has_todo())
 
+        # add ir.rule to prevent access on record
+        self.assertTrue(user0.has_group('base.group_user'))
+        model_id = self.env['ir.model'].search([('model', '=', record._name)]).id
+        rule = self.env['ir.rule'].create({
+            'model_id': model_id,
+            'groups': [(4, self.env.ref('base.group_user').id)],
```

Comment:
IIRC we can drop the `(4,X)` wrapping in x2m values when the meaning is actually `(6,0,[X])`. And simply specify `[X]`. We could start spreading the news, it's only been years :wink: